lib/pg_ha_migrations.rb

```diff
@@ -5,6 +5,16 @@
 require "relation_to_struct"
 
 module PgHaMigrations
+  Config = Struct.new(:disable_default_migration_methods)
```

Comment:
I'd like to see a couple of tests explicitly of the configuration setting and defaults.